app/geo/gui/src/ts/twxplore/gui/geo/reducers/map/mapReducer.ts

```diff
@@ -51,33 +57,39 @@ export const mapReducer = (state: MapState, action: BaseAction): MapState => {
         Loops throrugh the attributes of the feature, checks to see which are of type number
         and updates the min and maxes of the attribute in the filterState if neccessary
         */
-        const filterStateOfType =
-          result.featureTypesFilters[addedFeature.type!];
-        if (addedFeature.type === FeatureType.Transmission) {
-          for (const attribute of Object.keys(addedFeature)) {
-            //     console.log(attribute + " " + typeof ((addedFeature as any)[attribute]));
-            console.log(
-              FeatureAttributeName[
-                attribute as keyof typeof FeatureAttributeName
-              ]
-            );
-            if (getFeatureAttributeByName(attribute).isNumeric) {
-              {
-                if (
-                  (addedFeature as any)[attribute] <
-                  filterStateOfType[attribute].min!
-                )
-                  filterStateOfType[attribute].min = (addedFeature as any)[
-                    attribute
-                  ];
-                else if (
-                  (addedFeature as any)[attribute] >
-                  filterStateOfType[attribute].max!
-                )
-                  filterStateOfType[attribute].max = (addedFeature as any)[
-                    attribute
-                  ];
-              }
+        if (!result.featureTypesFilters[addedFeature.type!]) {
```

Comment:
You seem to be tracking multiple "per-type" states, too. You could collapse these into a single object e.g., FeatureTypeState { features; state enum (current needsFilters); max; ... } then have a map of type -> that. It's essentially your FilterState, but since filters are per-type (?) you're merging multiple things.